src/loaders/TextureLoader.js

```diff
@@ -22,11 +21,6 @@ class TextureLoader extends Loader {
 		loader.load( url, function ( image ) {
 
 			texture.image = image;
-
-			// JPEGs can't have an alpha channel, so memory can be saved by storing them as RGB.
-			const isJPEG = url.search( /\.jpe?g($|\?)/i ) > 0 || url.search( /^data\:image\/jpeg/ ) === 0;
-
-			texture.format = isJPEG ? RGBFormat : RGBAFormat;
```

Comment:
@Oletus FYI

Reply:
This seems like an acceptable compromise to ensure generateMipmap works, devices tend to have a lot more memory these days compared to when this change was made.